tests/test_projectbuilder.py

```diff
@@ -21,7 +21,7 @@
 
 DEFAULT_BACKEND = {
     'build-backend': 'setuptools.build_meta:__legacy__',
-    'requires': ['setuptools >= 40.8.0'],
+    'requires': ['setuptools >= 40.8.0', 'wheel'],
```

Comment:
This seems like a regression. Why is it needed? `wheel` was required on older `setuptools` versions, but shouldn't be anymore. If our minimum `setuptools` version does not yet include the fix to return `wheel` in `get_requires_for_build_wheel`, we should update it.